FrontEnd.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+import packageName.DataToFromComputeAPI;
+
+public class FrontEnd {
+    public static void main(String[] args) {
+        char delim = ' ';
+        Scanner in = new Scanner(System.in);
+        System.out.println("Enter the source type. Enter 1 for file, Enter 2 for manual input: ");
+        int inputType=in.nextInt();
+        
+        //Receive input from file
+        if(inputType==1) {
+            System.out.println("Enter the file path: ");
+            String path = in.next();
+            DataToFromComputeAPI data = new DataToFromComputeAPI();
+            List<Integer> list = data.readData(path);
+            computeResults(list);
+        } else if(inputType==2) {
```

Comment:
Remember to put else and if else statements on the same line as the ending curly brace, that should fix any checkstyle issues